feature/bgp/multihop/README.md

```diff
@@ -0,0 +1,81 @@
+# RT-1.63: BGP Multihop
+
+## Summary
+
+This test case validates the multihop eBGP feature, where two BGP speakers establish a peering session even when they are not directly connected.
+
+## Testbed type
+
+*  [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure 
+
+### Configuration
+
+1)  Create the topology below:
+
+```mermaid
+graph LR; 
+A[ATE:Port1] -- EBGP --> B[Port1:DUT:Port2];
+B ----> C[Port2:ATE] --> eBGP peer;
+```
+
+2)  Configure the eBGP peering session: Establish an eBGP peering relationship between ATE1 and Port 1 on the DUT.
+3)  Assign IP addresses: Configure IPv4 and IPv6 addresses on all relevant interfaces on the DUT, ATE1, and ATE2.
+4)  Create a loopback interface: Configure a loopback interface on the DUT and assign it an IP address. This will serve as the DUT's identifier for the BGP session.
+5)  Establish static routing on the DUT: Configure a static route on the DUT that directs traffic destined for the eBGP peer's IP address (connected to ATE2) through ATE2. This ensures the DUT can reach the peer.
+6)  Establish static routing on ATE2: Configure a static route on ATE2 that directs traffic destined for the DUT's loopback IP address towards the DUT. This allows the eBGP peer on ATE1 to reach and identify the DUT via its loopback address.
+
+### Tests
+
+### RT-1.63.1: Establish eBGP session over multihop
+
+1)  Configure the eBGP peering session: Establish an eBGP peering relationship between the DUT's loopback interface and the eBGP peer connected to ATE2. This means the DUT will use its loopback address as its BGP identifier.
+2)  Specify the neighbor address: Configure the DUT to use the eBGP peer's IP address (connected to ATE2) as the neighbor address for this BGP session.
+3)  Enable multihop: Since the eBGP peer is not directly connected to the DUT, enable the multihop option for this BGP neighbor on the DUT. Set the Time-to-Live (TTL) value to 2 to allow the BGP packets to traverse the intermediate link between the DUT and ATE2.
+4)  Validate session establishment: Confirm that the eBGP session is successfully established between the DUT's loopback interface and the eBGP peer connected to ATE2. This can be verified by checking the BGP neighbor status on both the DUT and the peer.
```

Comment:
```suggestion
4)  Validate session establishment: Confirm that the eBGP session is successfully established between the DUT's loopback interface and the eBGP peer connected to ATE2. This can be verified by checking the BGP neighbor status on both the DUT and the ATE.
```

Reply:
You'll need to specify the OC path for the DUT verification.